ui/app/mainui/AppMain.qml

```diff
@@ -2143,6 +2143,7 @@ Item {
                             navBar: appMain.navBar
                             isProduction: appMain.rootStore.isProduction
 
+                            activityCenterStore: appMain.activityCenterStore
```

Comment:
This property store assignment was missing... perhaps it would be much better to set all these properties to `required` @noeliaSD